net/table_net_dns_record.go

```diff
@@ -32,6 +32,7 @@ func tableNetDNSRecord(ctx context.Context) *plugin.Table {
 			{Name: "ip", Transform: transform.FromField("IP"), Type: proto.ColumnType_IPADDR, Description: "IP address for the record, such as for A records."},
 			{Name: "target", Type: proto.ColumnType_STRING, Description: "Target of the record, such as the target address for CNAME records."},
 			{Name: "priority", Type: proto.ColumnType_INT, Description: "Priority of the record, such as for MX records."},
+			{Name: "tag", Type: proto.ColumnType_STRING, Description: "An ASCII string that represents the identifier of the property represented by the record."},
```

Comment:
Is this column only for CAA records, or can it be found in others? If only for CAA, it should be noted in the description like other columns' descriptions.